backend-rust/src/graphql_api/account.rs

```diff
@@ -982,14 +982,14 @@ impl Account {
                     success,
                     events as "events: sqlx::types::Json<Vec<Event>>",
                     reject as "reject: sqlx::types::Json<TransactionRejectReason>"
-                FROM transactions
+                FROM transactions t
+                LEFT JOIN (
```

Comment:
This fixes the result, allowing the optimiser to only query the list once instead